.vscode/settings.json

```diff
@@ -17,5 +17,4 @@
   "files.watcherExclude": {
     "**/.fvm": true
   },
-  "java.configuration.updateBuildConfiguration": "automatic"
```

Comment:
Why is this here?

Reply:
It shouldn't be there thank you

Reply:
No? The line was there because it is useful, and you removed it from the file

Reply:
Can you please add the line back?